app/index.js

```diff
@@ -4,4 +4,5 @@ import App from './App.js';
 AppRegistry.registerComponent('app', () => App);
 
 // The below line is necessary for use with the TestBed App
+// CR * boilerplate
```

Comment:
Eventually you want to make it less obvious that this project was based on another boilerplate. This is about taking more comprehensive creative ownership of your repository! 💪